include/xlnt/cell/cell.hpp

```diff
@@ -651,6 +646,11 @@ class XLNT_API cell
 /// </summary>
 XLNT_API bool operator==(std::nullptr_t, const cell &cell);
 
+/// <summary>
+/// Returns true if this cell is uninitialized.
+/// </summary>
+XLNT_API bool operator==(const cell &cell, std::nullptr_t);
```

Comment:
Note: this operator is just the member operator moved out for consistency.
I was surprised by the presense of this operator. Comparing non-pointers against null is not something I would expect to be able to do. An is_empty/unused/... function would be more typical

Reply:
Opened #317 to clarify